app/src/androidTest/java/com/lightricks/feedexercise/data/FeedDaoTest.kt

```diff
@@ -0,0 +1,119 @@
+package com.lightricks.feedexercise.data
+
+import android.content.Context
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.room.Room
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.*
+import com.lightricks.feedexercise.database.FeedDao
+import com.lightricks.feedexercise.database.FeedDatabase
+import com.lightricks.feedexercise.database.FeedEntity
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Tests for [FeedDaoTest] class.
+ */
+@RunWith(AndroidJUnit4::class)
+class FeedDaoTest {
+
+    @get:Rule
+    val instantExecutorRule = InstantTaskExecutorRule() // TODO ask Nissim
+
+    private lateinit var feedDao: FeedDao
+
+    private lateinit var feedDatabase: FeedDatabase
+
+    private val context = ApplicationProvider.getApplicationContext<Context>()
+
+    companion object{
+        private val feedEntity1: FeedEntity = FeedEntity("1", "1", true)
+        private val feedEntity2: FeedEntity = FeedEntity("2", "2", true)
+        private val feedEntity3: FeedEntity = FeedEntity("3", "3", true)
+        private val feedEntity4: FeedEntity = FeedEntity("4", "4", true)
+
+        private val list1 = listOf(feedEntity1, feedEntity2)
+        private val list2 = listOf(feedEntity3, feedEntity4)
+        private val emptyList = emptyList<FeedEntity>()
+    }
+
+
+    @Before
+    fun setDatabase(){
+        feedDatabase = Room.inMemoryDatabaseBuilder(context, FeedDatabase::class.java)
+            .build()
+        feedDao = feedDatabase.feedDao()
+    }
+
+
+    @After
+    fun closeDatabase(){
+        feedDatabase.close()
+    }
+
+    @Test
+    fun testGetTableSize_initialTableSize_shouldReturn0(){
```

Comment:
Please remove the prefix `test` from all the method name. We already have this information that the method is a test with the `@Test` annotation.